emails/2023-04-07.md

```diff
@@ -0,0 +1,77 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+I know what you all are thinking, "Where was my Friday Tim email? I hope he's ok." 
+
+Well, I was on vacation last week so you get a Monday email instead of a Friday email. I appreciate all the concern even though I did not receive a single "Are you ok?" email. 
+
+### ChatGPT Integration
```

Comment:
I don't know if a hidden command line tool should be above the fold. Isn't AWS goodness going to be more likely to result in paying customers?

Reply:
I think it's just cooler.